flow/layers/image_filter_layer.cc

```diff
@@ -31,9 +31,6 @@ void ImageFilterLayer::Preroll(PrerollContext* context,
   if (!context->has_platform_view && context->raster_cache &&
       SkRect::Intersects(context->cull_rect, paint_bounds())) {
     SkMatrix ctm = matrix;
-#ifndef SUPPORT_FRACTIONAL_TRANSLATION
-    ctm = RasterCache::GetIntegralTransCTM(ctm);
-#endif
     context->raster_cache->Prepare(context, this, ctm);
```

Comment:
For cases like these (and I'm guessing there are a large number of them, the `ctm` local variable is unnecessary and should be deleted.